### PR TITLE
Documentation for release-2018-01

### PR DESCRIPTION
**Pull request by peyerluk**
#### Changes

- Updated docs for ContentTypes
- Added Design-Loader configuration options
- Added Elasticsearch version config

### CODE REVIEW

File: concepts/copying-documents/document_copy_feature.md [hunk 1]
```diff
@@ -1,40 +1,43 @@
 # Document Copy Feature
 
 ## Document Copy Basics
+
 ![copy-basics](https://cloud.githubusercontent.com/assets/172394/18782898/0beb0214-8189-11e6-98cc-e5e6728456de.png)
 ([original diagram ressource](https://www.draw.io/#G0B2rv2Pw26xPLT3hXQ3BsZU1lWlE))
 
-If you want to setup and configure an article copy, you need a rough understanding of channels, designs, layouts and components.
+If you want to setup and configure an article copy, you need a rough understanding of channels, contentTypes, designs, layouts and components.
```
Comment (gabrielhase) [inline concepts/copying-documents/document_copy_feature.md]: I wouldn't use camel casing in a written text, so `contentTypes` -> `content types`

File: concepts/copying-documents/document_copy_feature.md [hunk 2]
```diff
@@ -1,40 +1,43 @@
...
 
 Every `channel` has assigned a `livingdocs-design` (`D1, D2, ...`) and every design has one or more layouts (`L1, L2, ...`) with one ore more component (`C1, C2, ...`).
+
+For every layout there must be a contentType configuration in the channel.
```
Comment (gabrielhase) [inline concepts/copying-documents/document_copy_feature.md]: `... there must be a corresponding content type ...`

File: reference-docs/common-designs/design_config.md [hunk 3]
```diff
@@ -157,8 +157,8 @@ In a component the aspect ratios can be defined per image directive. In the foll
 
 ## Layouts
 
-Layouts define different type of documents. When creating a document in the
-editor, a user can choose which layout he wants to use.
+Layouts define different type of documents. For every layout there must be a corresponding `contentType` in the channel configuration of the server. When creating a document in the
```
Comment (gabrielhase) [inline reference-docs/common-designs/design_config.md]: `Layouts define subtypes of a channel or design. ... `

File: reference-docs/server-configuration/config.md [hunk 4]
```diff
@@ -193,22 +193,67 @@ emails: {
...
+#### Design Loader
 
-#### Designs
+The DesignLoader offers different options how designs are loaded. By default designs
+are loaded from the local database.
+
+Options include:
+* Load a specific design from a url
+* Preload a design into memory (useful for testing, or if you only change designs
+  during a deploy)
```
Comment (gabrielhase) [inline reference-docs/server-configuration/config.md]: `... deployment)`

File: reference-docs/server-configuration/config.md [hunk 5]
```diff
@@ -193,22 +193,67 @@ emails: {
...
+* Load a design from another Livingdocs Server
+
+All these options can work alongside each other.
 
-Configure the Livingdocs Design Server.
+If a design is requested the design loader fill do in this order:
```
Comment (gabrielhase) [inline reference-docs/server-configuration/config.md]: `... the design loader will execute the steps in the given order:`

File: reference-docs/server-configuration/content-type-config.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
+# Content Type Configuration
+
+You can define as many contentTypes per channel as you want. A contentType represents
+a specific kind of document. The contentTypes define which components can be used in
+a document and which metadata are present.
+
+Whenever you create a document you have to choose of what contentType it will be. The contentType of a document cannot be changed after creation.
+
+The `documentType` distinguished between articles and pages. Articles will be accessible in the main dashboard in the editor. Pages appear on a different screen. It is also possible to give distinct permissions for articles or pages.
```
Comment (gabrielhase) [inline reference-docs/server-configuration/content-type-config.md]: `distinguished` -> `distinguishes`

File: reference-docs/server-configuration/content-type-config.md [hunk 7]
```diff
@@ -0,0 +1,139 @@
...
+
+*Important to know* For every `contentType` there must be a `layout` in the design.
+The name layout of the layout must be exactly the same as the `contentType.handle`.
```
Comment (gabrielhase) [inline reference-docs/server-configuration/content-type-config.md]: ?

**Comment by DaRaFF** [comment]
@peyerluk I will merge this PR. The people want already to check the doc for content-types. Please create another PR, if you want to extend the doc.

**Pull request merged**